client/src/app/map/map.component.ts

```diff
@@ -131,11 +132,27 @@ export class MapComponent implements OnInit {
         for (var i = 0; i < this.preparednessLayers.length; i++){
             var newSource = new ol.source.TileWMS({
                 params: {
-                    'LAYERS': this.preparednessLayers[i].layerName,
+                    'LAYERS': 'dmis:' + this.preparednessLayers[i].layerName,
```

Comment:
Agreed with @hunt3ri that we'll do this at API level. I'll merge this PR and update when the API change has been made.